share/spice/expand_url/expand_url.js

```diff
@@ -21,12 +22,16 @@
                 sourceUrl: "http://untiny.me/",
                 sourceName: "Untiny"
             },
+            normalize: function (item) {
+                return {
+                    title: item.org_url,
+                    url: item.org_url,
+                    description: "Expanded URL for: " + decodeURIComponent(query)
```

Comment:
Thanks! :heart_eyes: 

That will give a good insight into each of the templates. Let me know if we are good to go with making these changes to the templates to bring same features to all of them. I'd love to take that up! :santa: